llvm/docs/DeveloperPolicy.rst

```diff
@@ -476,11 +476,12 @@ Obtaining Commit Access
 -----------------------
 
 We grant commit access to contributors with a track record of submitting high
-quality patches.  If you would like commit access, please send an email to
-`Chris <mailto:clattner@llvm.org>`_ with your GitHub username.  This is true
-for former contributors with SVN access as well as new contributors. If
-approved, a GitHub invitation will be sent to your GitHub account. In case you
-don't get notification from GitHub, go to
+quality patches.  If you would like commit access, please use this `link
+<https://github.com/llvm/llvm-project/issues/new?title=Request%20Commit%20Access%20For%20<user>&body=%23%23%23%20Why%20Are%20you%20requesting%20commit%20access%20?>`_ to file
+and issue and request commit access.  Replace the <user> string in the title
```

Comment:
Should we ask for the list of merged PRs to simplify the review?

Reply:
Would the link get too complicated if we tried to link back to this document?

Reply:
> Should we ask for the list of merged PRs to simplify the review?

This is not currently required, and I'm trying to be careful not to change the commit access requirements in this PR.  I just want to change the process.

Reply:
> Would the link get too complicated if we tried to link back to this document?

It's possible, can you give me an example of what you think the full issue body should look like?

Reply:
> This is not currently required, and I'm trying to be careful not to change the commit access requirements in this PR. I just want to change the process.

Well, the policy is to have quality patches submitted. So, how we'd verify this? See e.g. https://github.com/llvm/llvm-project/issues/100457#issuecomment-2248855192

Reply:
@asl I know that's what the documentation says, but that's not what's been happening in practice.  There has been a lot of disagreement over what the commit access requirements should be and I don't want to restart this discussion just yet.  I want to improve the process, but also keep the status quo as far as commit access requirements for now.

Reply:
I think in any case if the reviewer wants to get this piece of data it’s a fairly easy search on GitHub UI.